metriq_gym/benchmarks/benchmark.py

```diff
@@ -1,17 +1,30 @@
 import argparse
-from typing import Any
+from dataclasses import dataclass
 
-from metriq_gym.job_manager import JobManager
+from pydantic import BaseModel
```

Comment:
I don't see `BaseModel` in the diff below. Maybe not, but can this import be removed?

Reply:
BaseModel is the type of params in the BenchmarkData constructor